channeldb/channel.go

```diff
@@ -2550,6 +2550,8 @@ type HTLC struct {
 	// HTLC. It is stored in the ExtraData field, which is used to store
 	// a TLV stream of additional information associated with the HTLC.
 	BlindingPoint lnwire.BlindingPointRecord
+
+	CustomRecords lnwire.CustomRecords
```

Comment:
nit: missing Godoc comment. And maybe we should mention here that this will be encoded into the `ExtraData` alongside the `BlindingPoint` when encoded (for sending on the wire or storing to the DB).